.travis.yml

```diff
@@ -7,14 +7,14 @@ os:
   - osx
```

Comment:
a question for the maintainers - but osx seem overkill here

Reply:
great feedback! committed changes. lets see what the maintainers say about 
```suggestion
  - osx
```